parsl/monitoring/db_logger.py

```diff
@@ -30,23 +32,85 @@ def emit(self, record):
         pass
 
 
-def get_db_logger(host='search-parsl-logging-test-2yjkk2wuoxukk2wdpiicl7mcrm.us-east-1.es.amazonaws.com',
-                  port=443,
-                  enable_es_logging=False,
-                  index_name="my_python_index",
-                  version='1.0.0',
+class LoggerConfig():
+    """ This is a config class for creating a logger. """
+    def __init__(self,
+                 host='search-parsl-logging-test-2yjkk2wuoxukk2wdpiicl7mcrm.us-east-1.es.amazonaws.com',
```

Comment:
is that a useful default? or should the code throw an exception saying this parameter is mandatory when needed?

(alternatively: aree arbitrary parsl users expected to often use that particular instance? and if so, get a proper DNS name set up for it)

Reply:
I agree this is a bad choice for a default. I have made some changes to these defaults and tried to raise errors for some incorrect configurations. What level of checking would you recommend for these inputs?